internal/pkg/cli/env_init.go

```diff
@@ -241,6 +242,10 @@ func (o *initEnvOpts) Execute() error {
 		return fmt.Errorf("get environment struct for %s: %w", o.name, err)
 	}
 	env.Prod = o.isProduction
+	env.CustomConfig = config.CustomConfig{
```

Comment:
nit: What do think of making this a pointer? So that it's not added to the JSON body if there is no customization.
```go
var customConf *config.CustomConfig
if i, a := o.importVPCConfig(), o.adjustVPCConfig(); i != nil && a != nil {
	customConf := &config.CustomConfig{i, a}
}
```